README.md

```diff
@@ -1 +1,31 @@
-# cdnvalidator
+# cdnvalidator service
+
+[![Build Status](https://drone.corp.mongodb.com/api/badges/kanopy-platform/cdnvalidator/status.svg)](https://drone.corp.mongodb.com/kanopy-platform/cdnvalidator)
+
+The cdnvaildator provides multi-tenant entitlement abstraction layer into CloudFront invalidations which may have different owners, performance, and cache control requirements.
```

Comment:
nit: spelling, and I think this flows a bit better to breaks up the terse sentence:

```suggestion
The cdnvalidator provides an abstraction layer for CloudFront invalidations. It provides control for performance, different owners of distribution paths, and cache control requirements.
```

Reply:
I took most of this, but changed some of it.